marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command.hs

```diff
@@ -54,10 +48,8 @@ data Command
 -- | Read options from the environment and the command line.
 getOptions :: IO Options
 getOptions = do
-  syncHostParser <- optParserWithEnvDefault O.syncHost
-  syncSyncPort <- optParserWithEnvDefault O.syncSyncPort
-  txHostParser <- optParserWithEnvDefault O.txHost
-  txCommandPortParser <- optParserWithEnvDefault O.txCommandPort
+  hostParser <- optParserWithEnvDefault O.runtimeHost
+  portParser <- optParserWithEnvDefault O.runtimePort
```

Comment:
These two appear to be in [Language.Marlowe.Runtime.CLI.Option](https://github.com/input-output-hk/marlowe-cardano/blob/scp-5010-cli-proxy/marlowe-runtime/config/Language/Marlowe/Runtime/CLI/Option.hs) but `syncHost`, `syncSyncPort`, `txHost`, and `txCommandPort` are still in there. Should they be removed?

Reply:
That module's name didn't end up aging well past its original intent. It's like the standard set of option parsers so that we get consistent option names / env vars across the runtime and clients.

Reply:
Ok, no problem.